### PR TITLE
[1LP][RFR] New Test: Testing automate domain import via rake command

### PR DESCRIPTION
**Pull request by ganeshhubale**
## Purpose or Intent
- This PR automates the scenario of importing automate domain via rake command using git repo and checking via UI if imported domain is exists without any error.

### PRT Run
{{ pytest: cfme/tests/automate/test_git_import.py::test_automate_git_import_case_insensitive --use-template-cache -qsvvv }}

### CODE REVIEW

File: cfme/tests/automate/test_git_import.py [hunk 1]
```diff
@@ -352,3 +352,28 @@ def _cleanup():
         domain.delete_if_exists()
 
     assert automate_domain.exists
+
+
+@pytest.mark.tier(1)
+def test_automate_git_import_case_insensitive(request, appliance):
+    """
+    bin/rake evm:automate:import PREVIEW=false
```
Comment (valaparthvi) [inline cfme/tests/automate/test_git_import.py]: It would be nicer if you put this into `testSteps`.

File: cfme/tests/automate/test_git_import.py [hunk 2]
```diff
@@ -352,3 +352,28 @@ def _cleanup():
...
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/8h
+        tags: automate
+        startsin: 5.7
+    """
+    appliance.ssh_client.run_rake_command(
+        "evm:automate:import PREVIEW=false GIT_URL=https://github.com/ganeshhubale/SimpleDomain "
```
Comment (izapolsk) [inline cfme/tests/automate/test_git_import.py]: @ganeshhubale, your test looks good but I doubt @mshriver will approve using your own repo. Could you use one of our official repos for that  instead ?

Comment (ganeshhubale) [reply cfme/tests/automate/test_git_import.py]: @izapolsk Yes I agree with you.
So I am trying this test case with repository from ```RedHatQE``` group.
Repository URL: https://github.com/RedHatQE/ManageIQ-automate-git

**Comment by izapolsk** [comment]
@ganeshhubale, please check PRT

**Pull request merged**